### PR TITLE
Design

### PR DESCRIPTION
**Pull request by kosssi**
We can display new design with `flag('transaction-history', true)`

## Screenshots

Without flag | with flag
-------------|---------
capture d ecran 2018-12-27 a 10 37 51 | capture d ecran 2018-12-27 a 10 36 36
capture d ecran 2018-12-27 a 10 37 40 | capture d ecran 2018-12-27 a 10 37 17

### CODE REVIEW

File: src/ducks/transactions/TransactionHeader.jsx [hunk 1]
```diff
@@ -125,7 +124,9 @@ class TransactionHeader extends Component {
       <Header {...colorProps} fixed>
         <Padded className={isMobile ? 'u-p-0' : ''}>
           {this.displayAccountSwitch()}
-          {this.displayBalanceHistory()}
+        </Padded>
+        {this.displayBalanceHistory()}
+        <Padded className={isMobile ? 'u-p-0' : ''}>
```
Comment (drazik) [inline src/ducks/transactions/TransactionHeader.jsx]: Nit: I find `cx({ 'u-p-0': isMobile })` more readable.

File: src/ducks/balance/History.styl [hunk 2]
```diff
@@ -1,9 +1,11 @@
 .HistoryChart
-    padding-bottom .5rem
     overflow-y auto
     background-color dodgerBlue
     margin 0
 
+.History .HistoryChart
+    padding-bottom .5rem
```
Comment (drazik) [inline src/ducks/balance/History.styl]: Why moving this property in another selector? Is the goal to add more specificity?

File: src/ducks/transactions/TransactionHeader.jsx [hunk 3]
```diff
@@ -126,7 +126,7 @@ class TransactionHeader extends Component {
 
     return (
       <Header {...colorProps} fixed>
-        <Padded className={isMobile ? 'u-p-0' : ''}>
+        <Padded className={isMobile ? 'u-p-0' : withChart ? 'u-pb-0' : ''}>
```
Comment (drazik) [inline src/ducks/transactions/TransactionHeader.jsx]: I propose to use `cx` here to make it more readable:

```
cx({
  'u-p-0': isMobile,
  'u-pb-0': !isMobile && withChart
})
```

File: src/components/SelectDates/SelectDates.styl [hunk 4]
```diff
@@ -11,24 +11,74 @@ select-date-height-small = 2.75rem
...
+        color slateGrey
+
+        .SelectDates__chip
+            height select-date-height-small
+
 .SelectDatesColor_primary
+    border-bottom 0
+
+    .SelectDates__separator
+        border-left: 0.063rem solid var(--primary)
```
[the remark this answers is not in the document]
Comment (kosssi) [reply src/components/SelectDates/SelectDates.styl]: I do not know where it comes from 😮

File: src/ducks/balance/History.styl (hunk 2 above)
Comment (kosssi) [reply src/ducks/balance/History.styl]: Yes because HistoryChart can be use on Balance & Transaction page

Comment (drazik) [reply src/ducks/balance/History.styl]: Then maybe this padding should not be managed by `History`, but by the component that uses `History`. By passing a custom `className` to it for example.

**Pull request merged**